dags/databricks-ml-example.py

```diff
@@ -0,0 +1,85 @@
+from datetime import datetime
+import logging
+
+from airflow.decorators import task, dag
+from airflow.providers.databricks.hooks.databricks import DatabricksHook
+from airflow.providers.databricks.operators.databricks import DatabricksSubmitRunOperator
+# from astronomer.providers.databricks.operators.databricks import DatabricksSubmitRunOperatorAsync
+
+import mlflow
+
+docs = """
+Demonstrates orchestrating ML pipelines executed on Databricks with Airflow
+"""
+
+@dag(
+    start_date=datetime(2022, 1, 1),
+    schedule_interval=None,
+    catchup=False,
+    doc_md=docs
+)
+def databricks_ml_example():
+
+    ingest_notebook = {
+        'notebook_path': '/Users/<your username>/<BigQuery-to-Databricks-notebook>',
+    }
+
+    ingest = DatabricksSubmitRunOperator(
+        task_id='ingest_notebook_task',
+        existing_cluster_id='0224-221140-suj0ngd4',
```

Comment:
Can throw the existing_cluster_id into a global variable. It's mentioned a handful of times.